frappe/public/js/frappe/list/listview.js

```diff
@@ -61,7 +61,9 @@ frappe.views.ListView = Class.extend({
 		// add workflow field (as priority)
 		this.workflow_state_fieldname = frappe.workflow.get_state_fieldname(this.doctype);
 		if(this.workflow_state_fieldname) {
-			add_field(this.workflow_state_fieldname);
+			if (!frappe.workflow.workflows[this.doctype]["override_status"]) {
```

Comment:
what if there is no workflow? @RobertSchouten best to cover all cases. Will fix this.